manual/src/server/sal.md

```diff
@@ -0,0 +1,789 @@
+- Cancel the discource deployment
+
+```
+func (c *Client) CancelTaigaDeployment(name string) error 
+```
+
+## Stellar
+
+### Load
+- loads key
+- chooses network
+- starts session and creates state for client
+
+```
+func (c *Client) Load(args Load) error
+```
+
+### CreateAccount
+- Creates account on stellar given the network
+
+```
+func (c *Client) CreateAccount(network string) (string, error)
+```
+
+### Address
+- Get the stellar public address of the loaded secret
+
+```
+func (c *Client) Address() (string, error)
+```
+
+### Transactions
+- Get the last transactions of your account
+
+```
+func (c *Client) Transactions(args Transactions) ([]horizon.Transaction, error)
+```
+
+### AccountData
+- Get data related to a stellar account, leave account empty for account data of loaded account
+
+```
+func (c *Client) AccountData(account string) (horizon.Account, error)
+```
+
+### Swap
+- Swap tokens from one asset to the other (xlm to tft, etc)
+
+```
+func (c *Client) Swap(args Swap) error
+```
+
+### Transfer
+- Transfer tokens from one account to the other on stellar chain
+
+```
+func (c *Client) Transfer(args Transfer) (string, error)
+```
+
+### Balance
+- Get the balance of a specific address
+- If address is empty the balance of the loaded account will be returned
+
+```
+func (c *Client) Balance(address string) (string, error)
+```
+
+### BridgeToEth
+- Bridge tokens of your stellar account to an account on ethereum
+
+```
+BridgeToEth(args BridgeTransfer) (string, error)
+```
+
+### BridgeToTfchain
+- Bridge tokens of your stellar account to an account on tfchain (provide twinid)
+
+```
+func (c *Client) BridgeToTfchain(args TfchainBridgeTransfer) (string, error)
+```
+
+### AwaitBridgedFromEthereum
+- Await till a transaction is processed on ethereum bridge that contains a specific memo, that transaction is the result of a bridge from ethereum to the loaded stellar account
+
+```
+func (c *Client) AwaitTransactionOnEthBridge(memo string) error
+```
+
+## Ethereum
+
+### Load
+- loads key
+- chooses network
+- starts session and creates state for client
+
+```
+func (c *Client) Load(args Load) error
+```
+
+### Balance
+- Balance of an address, if address is empty the balance of the loaded account will be returned                                    
+- Provide the asset to get the balance of (eth, tft)
+
+```
+func (c *Client) Balance(args Balance) (string, error)
+```
+
+### Height
+- Height of the chain for the connected rpc remote
+
+```
+func (c *Client) Height() (uint64, error)
+```
+
+### Transfer
+- Transer an amount of Eth from the loaded account to the destination. The transaction ID is returned.
+
+```
+func (c *Client) Transfer(args Transfer) (string, error)
+```
+
+### Quote
+- quote a swap of asset
+
+```
+func (c *Client) Quote(args Quote) (string, error)
+```
+
+### Swap
+- Swap tokens from one asset to the other (eth to tft, etc)
+
+```
+func (c *Client) Swap(args Swap) (string, error)
+```
+
+### TransferEthTft
```

Comment:
This is confusing, we are already on ETH network, should be TransferTFT imo

Reply:
Ok. Yes I meant here that it transfers TFTs (that are on the ethereum network). Maybe I should incorporate this into the Transfer function and add an extra argument (asset)?

Reply:
Hmm for a SAL, this makes the most sense indeed to have a shortcut directly like you did indeed,   and reduce the mental overhead for the user, but definitely @despiegk can help out deciding on this